src/wp-includes/class-wp-widget.php

```diff
@@ -152,7 +152,7 @@ public function form( $instance ) {
 	 *
 	 * @since 2.8.0
 	 *
-	 * @param string $id_base         Optional. Base ID for the widget, lowercase and unique. If left empty,
+	 * @param string $id_base         Optional Base ID for the widget, lowercase and unique. If left empty,
```

Comment:
```suggestion
	 * @param string $id_base         Base ID for the widget, lowercase and unique. If left empty,
```
I'd consider removing "Optional" altogether from this description and from `__construct()` below.

Reply:
Done.